project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=0.13.12
```

Comment:
That fixed it.  We got the intermittent client failure again, but not the persistent sbt crash.